### PR TITLE
Fix(eos_cli_config_gen): Do not render entries with only sequence number from ip_access_list

### LINKED ISSUE #4423: Fix(eos_cli_config_gen): Do not render entries with only sequence number and nothing else from ip_access_list
### Issue Summary

Reported from the field

This input

```yaml
ipv4_acls:
  - name: WAN_ACL_IN_Et1
    entries:
      - source: any
        destination: any
        sequence: 10
        action: permit
```

Will generate the following invalid config:

```
ip access-list WAN_ACL_IN_Et1
   10 
!
```

Because the `eos_designs` `ipv4_acls` key will pass along the entry untouched to the underlying `ip_access_list` model.

## Expected resolution

Please make sure no sequence number is generated when both `remark` and `protocol` are missing.
The problem is in this template where the acl entry string is ALWAYS rendered https://github.com/aristanetworks/avd/blob/devel/python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2#L133 even if neither the `if` https://github.com/aristanetworks/avd/blob/devel/python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2#L34 nor the `elif` https://github.com/aristanetworks/avd/blob/devel/python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2#L38 are matched.

In this scenario, no sequence number should be rendered

### Which component(s) of AVD impacted

eos_cli_config_gen

### How do you run AVD ?

Ansible CLI (with virtual-env or native python)

### Steps to reproduce

```shell
cf example above or use directly the eos_cli_config_gen model.
```

### Relevant log output

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by gmuloc**
## Change Summary

Skip issues if `remark` or any field in `[protocol, action, source, destination]` is missing 

## Related Issue(s)

Fixes #4423

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

Test if the entry is empty, if it is skip rendering it (and the sequence number)

## How to test

molecule - first commit demonstrate the issue, second solves it

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2 [hunk 1]
```diff
@@ -130,7 +126,14 @@ ip access-list {{ acl.name }}
 {%                     endif %}
 {%                 endif %}
 {%             endif %}
+{#             - only render if the string is not empty #}
```
Comment (ClausHolbechArista) [inline python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2]:
```suggestion
{#             Only render if the string is not empty #}
```

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2 [hunk 2]
```diff
@@ -130,7 +126,14 @@ ip access-list {{ acl.name }}
...
+{%             if acl_entry.string != "" %}
+{#                 - prepend sequence number if defined and space #}
```
Comment (ClausHolbechArista) [inline python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2]:
```suggestion
{#                 Prepend sequence number if defined #}
```

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2 (hunk 1 above)
Comment (gmuloc) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2]: I was matching the rest of the file with the `-` do you want to change line 28 and 26 as well?

Comment (ClausHolbechArista) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2]: It is not a style we usually have. Feel free to update elsewhere.

**Review by Vibhu-gslab — APPROVED** [review]
LGTM!

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2 (hunk 1 above)
Comment (gmuloc) [reply python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2]: this was done in last commit already

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**